frontend/src/pages/project/Settings/index.vue

```diff
@@ -40,6 +40,7 @@ export default {
                 { name: 'Environment', path: './environment' }
             ]
             if (this.hasPermission('project:edit')) {
+                this.sideNavigation.push({ name: 'Stages', path: './stages' })
```

Comment:
GitHub talks about environments: https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment

Reply:
Azure & IBM Cloud talk about Stages: Azure refer to a step in the Pipeline as a "Stage", as does IBM Cloud - https://learn.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml